authentik/lib/expression/evaluator.py

```diff
@@ -216,6 +220,74 @@ def expr_create_jwt(
         access_token.save()
         return access_token.token
 
+    def expr_send_email(
+        self,
+        address: str,
+        subject: str,
+        body: str | None = None,
+        stage: "EmailStage | None" = None,
+        template: str | None = None,
+        context: dict | None = None,
+    ) -> bool:
+        """Send an email using authentik's email system
+
+        Args:
+            address: Email address to send to
```

Comment:
Is there any way to allow for setting multiple `To` headers, and also `Cc` and `Bcc`?

Reply:
@rissson to support `cc` and `bcc` we should modify `authentik.stages.email.utils.TemplateEmailMessage` and I think it's out of scope for this feature
but multiple `To` I think is possible with the current implementation, I'll look into it